### PR TITLE
Added CH reserved properties info from KB article

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Comment by robday-sf** [comment]
Looks good to me. Thanks!

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 1]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
 For more information about nondefault property placeholder options, see the https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.html[Spring documentation on Property Placeholder options].
 
+== CloudHub Reserved Properties 
+
+CloudHub reserves a set of properties and prevents apps from overwriting the values of those properties.
+If an application attempts to assign a value to a reserved property, CloudHub doesn't use the assigned value.
+Instead, it returns the value as set internally without throwing an error or exception.
+
+Applications can reference reserved properties, for example, to retrieve the application name or deployment region.
+You can use the `${csorganization.id}` and `${environment.id}` properties to retrieve those values if your app makes Anypoint Platform API calls.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]:
```suggestion
Use the `${csorganization.id}` and `${environment.id}` properties to retrieve those values if your app makes Anypoint Platform API calls.
```

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 2]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+You can use the `${worker.id}` to identify each worker in a multi-worker application.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]:
```suggestion
Use `${worker.id}` to identify each worker in a multiworker application.
```

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 3]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+
+--
+[NOTE]
+The `${java.version}` property value doesn't include the point release for the JDK version.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: I stumbled on "point release," but figured it out.
I guess I am used to "dot release"?
Can we instead use the release number, to be exact?

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 4]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+Additionally, CloudHub runtime release updates might upgrade the JDK.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: Is it obvious how this second sentence is "additional" to the previous? 
It isn't to me, but...that's just me....   ;-)

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 5]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+|`${domain}`|App name|`ch-https-41-demo`
+|`${fullDomain}`|Full domain name of app|`ch-https-41-demo.au-s1.cloudhub.io`
+|`${application.aws.region}`|App deployment region|`ap-southeast-2`
+|`${http.port}`|HTTP port|8081
+|`${https.port}`|HTTPS port|8082
+|`${http.private.port}`|HTTP private port|8091
+|`${https.private.port}`|HTTPS private port|8092
+|`${mule.home}`|Absolute path to Mule installation|`/opt/mule/mule-4.1.1`
+|`${spring.profiles.active}`|Used internally, *do not use*|`*****`
+|`${logging.*}`|Used internally, *do not use*|`*****`
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: I wonder if we have to include it, or whether we should more specifically call this out as not to be used?
I worry about stuff getting lost in a table....

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 6]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+|`${host}`|Used internally for local IP address of a worker|`10.40.122.33`
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]:
```suggestion
|`${host}`|Used internally for the local IP address of a worker|`10.40.122.33`
```

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 7]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+|`${anypoint.url}`|Used internally as the base Anypoint Platform URL|US: `\https://anypoint.mulesoft.com`
+
+GovCloud: `\https://gov.anypoint.mulesoft.com`
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: Was this intended to be commented out?
What is it?
GovCloud > MuleSoft Government Cloud

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 8]
```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
...
+|===
+
+--
+[NOTE]
+For propriety reasons, MuleSoft doesn't make public all internal, reserved properties.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: propriety > proprietary?

Comment (kbostrom) [reply modules/ROOT/pages/cloudhub-manage-props.adoc]: Does this go without saying?

Comment (kbostrom) [reply modules/ROOT/pages/cloudhub-manage-props.adoc]: Also, ditto re. how second sentence is "additional" to first.

**Review by kbostrom — DISMISSED** [review]
Edited according to request

File: modules/ROOT/pages/cloudhub-manage-props.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/cloudhub-manage-props.adoc]: It changes so, no.

**Review by MichelleZMY — DISMISSED** [review]

**Review by kbostrom — COMMENTED** [review]
Reapproved without review

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**